source/SkiaSharp.Svg/SkiaSharp.Svg.Shared/SKSvg.cs

```diff
@@ -193,6 +193,23 @@ private void ReadElement(XElement e, SKCanvas canvas, SKPaint stroke, SKPaint fi
 			// parse elements
 			switch (elementName)
 			{
+				case "image":
+					{
+						var x = ReadNumber(e.Attribute("x"));
+						var y = ReadNumber(e.Attribute("y"));
+						var width = ReadNumber(e.Attribute("width"));
+						var height = ReadNumber(e.Attribute("height"));
+						var bytes = ReadBytes(e.Attributes().FirstOrDefault((arg) => arg.Name.LocalName == "href"));
+						var data = SKData.CreateCopy(bytes);
+						SKImage image = SKImage.FromEncodedData(data);
+						var rect = SKRect.Create(x, y, width, height);
+						canvas.DrawImage(image, rect);
+
+						data?.Dispose();
+						image?.Dispose();
```

Comment:
Maybe wrap the `data` and `image` objects in a `using` statement - just in case of an exception.